process/transactionEvaluator/transactionEvaluator.go

```diff
@@ -103,7 +103,7 @@ func (ate *apiTransactionEvaluator) ComputeTransactionGasLimit(tx *transaction.T
 	switch txTypeOnSender {
 	case process.SCDeployment, process.SCInvoking, process.BuiltInFunctionCall, process.MoveBalance:
 		return ate.simulateTransactionCost(tx, txTypeOnSender)
-	case process.RelayedTx, process.RelayedTxV2:
+	case process.RelayedTx, process.RelayedTxV2, process.RelayedTxV3:
 		// TODO implement in the next PR
```

Comment:
let's add a task for this on Jira - a long forgotten TODO.

Reply:
[added](https://multiversxlabs.atlassian.net/browse/MX-14626)